### DISCUSSION TITLE
Syser Kernel debugger?

### DISCUSSION THREAD

**Discussion by Imperatorn** [yanfengwu-syser/syserdebugger#2] [General]
Is this the syser kernel debugger or user mode debugger?

**Comment by jls28** [comment]
It seem
the visualstudio solution named Syser.sln is the user mode debugger and the
visualstudio solution named SyserApp.sln is the kernel mode debugger.
These two solutions share a lot of code.

I think the kernel debugger is not totally complete
because of 2 x DeviceIOControl with unknown IOCTL_  (file SyserInterface.cpp).
